docs/getting_started.md

```diff
@@ -4,44 +4,51 @@
 
 - Fill in missing data via imputation
 - Train and compare models based on your data
-- Save a model to produce daily predictions
-- Write predictions back to a database or csv file
-- Learn what factor drives each prediction
+- Save a model to produce daily or batch predictions
+- Write daily or batch predictions back to a database or csv file
+- Learn which factors drive each prediction
 
 ## Installation
 
 ### Windows
 
 - If you haven't, install 64-bit Python 3.5 via [the Anaconda distribution](https://www.continuum.io/downloads)
```

Comment:
Should be Python 3.6 (and I had problems loading healtcareai with Python 3.6.0-0 which went away when I upgraded to 3.6.1-2)